README.md

```diff
@@ -1,5 +1,5 @@
 <div align="center">
-  <img alt="QuestDB Logo" src="https://questdb.io/img/questdb-logo-themed.svg" width="305px"/>
+  <a href="https://questdb.io/" target="blank"><img alt="QuestDB Logo" src="https://questdb.io/img/questdb-logo-themed.svg" width="305px"/>
```

Comment:
```suggestion
  <a href="https://questdb.io/" target="blank"><img alt="QuestDB Logo" src="https://questdb.io/img/questdb-logo-themed.svg" width="305px"/></a>
```

Reply:
there are plenty of examples in the same file of correct `a` usage. Yes, you can leave them open, but this is not a well-formed HTML.

![image](https://user-images.githubusercontent.com/7276403/199218620-a89d2dda-da74-4db3-be45-0f685aac0ccc.png)

Reply:
in the above example `<A>` doesn't have `body` so it is self-closed `<A/>`. Here is example of `<A>` having `body`:

![image](https://user-images.githubusercontent.com/7276403/199218923-c4035080-e35a-4232-abf2-bb3755fd5926.png)

Reply:
Sorry for the confusion, I modified it as desired.